src/renderers/testing/ReactTestRendererFiberEntry.js

```diff
@@ -285,14 +285,19 @@ function toJSON(inst: Instance | TextInstance): ReactTestRendererNode {
   }
 }
 
-function nodeAndSiblingsArray(nodeWithSibling: ?Fiber) {
+function nodeAndSiblingsTrees(nodeWithSibling: ?Fiber) {
```

Comment:
We want to avoid optional types like `?Fiber`) in favor of `Fiber | null`. Since this file is already using optional types it's fine, just letting you know for the future.